articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,109 @@
+<!--more-->
+### Introduction
+Machine learning and deep learning are among the fastest-growing applications. Artificial Intelligence ([AI](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp)) is one of the fast-growing technologies that has significantly contributed to the growth of machine and deep learning.
+
+The AI incorporates and helps power the intelligence that is used to enhance decision making within both applications. These applications have contributed significantly in solving problems that previously affected performances in some applications.  
+
+New ideas in the tech world have led to various innovations that have sparked the nature of things. Alexa and Siri are [virtual assistant](https://www.investopedia.com/terms/v/virtual-assistant.asp) that make extensive use of machine learning and are widely used today.
+
+Machine learning and deep learning have attracted significant attention over the years. The popularity of these two has led to various applications implementing its tools.
+
+This article will focus on understanding what machine learning and deep learning are. By the end of the article the reader will have a deeper understanding on how these applications differ and enhances AI functionalities.
+
+### Machine Learning and Artificial Intelligence
+Machine learning is a [subset](https://en.wikipedia.org/wiki/Subset) of AI that allows computer algorithms to make accurate data outcomes through interpretations.
+
+Machine learning allows the use the AI towards automating several functionalities used by machine learning. Besides this, the AI enhances neuroscience that emulates human intelligence, an aspect that machine learning lacks.
+
+Several [synthetic intelligence](https://www.wikiwand.com/en/Synthetic_intelligence) aspects and modern [virtual assistants](https://www.investopedia.com/terms/v/virtual-assistant.asp) are created to achieve this.
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. These techniques include the use of [conventional algorithms](http://wiki.cas.mcmaster.ca/index.php/Conventional_Encryption_Algorithms) used in performing mathematical computation and predictions.
+
+Structured data is presented in the form of columns and rows. Examples of structured data include strings, dates and numbers.
+
+Unstructured data information is stored as files. Examples include; images, word documents, email files, videos and audio files.
+
+Machine learning uses both structured and unstructured data. The use of Support Vector Machines (SVMs) and decision-trees are examples that use unstructured data to make interpretations.
+
+Conventional algorithms aids in achieving data mining and predictive analytics in a search. A conventional algorithm is an algorithm that aid in identifying, deciphering, and enciphering keys that are identical, and need to be kept secret.
```

Comment:
This sentence was difficult to read

Reply:
I have made some modifications to enhance readability.